src/version-2-1-0.md

```diff
@@ -1,19 +1,30 @@
 Date: 2016-03-09
 Title: Release 2.1.0
-Tags: news, release notes
+Tags: news, release note
 Slug: uniqush-push-2-1-0
 Category: news
 
 Please check out our [release notes] for version 2.1.0.
 
 We've added new APIs for listing the subscriptions of a subscriber and for
-listing the services that exist.
+showing the configurations of the push service providers for services that
+exist.
 
 Concurrency issues in ADM and APNS have been fixed. The APNS implementation
-has been changed to use a new, reliable worker pool implementation. This
-implementation has been running at production scale at if(we) serving push
+has been changed to use a new, reliable worker pool implementation.
+
+This implementation has been running at production scale at if(we) serving push
 notifications for Tagged/hi5 for a while now and has been proven to be
-reliable.
+reliable. We have 12 processes handling a peak of 60 APNS push notifications
+per second, and the tier is overprovisioned in order to be able to deal with
+traffic increases and potential surges. That's about 5 APNS push notifications
+per process per second. In addition, we send about 270 GCM push notifications
```

Comment:
The first time it's referenced, "if(we)" -> a company named if(we). Possibly italicize.


"We have" -> "The company has"/"The company, which a uniqush maintainer works at,"

Write in third person, since "We" would probably be interpreted as the collective authors of uniqush by readers of a changelog. (Above, **We've** added new APIs for listing the subscriptions of a subscriber )